prototypes/technology/trainassembly-technology.lua

```diff
@@ -0,0 +1,151 @@
+require "util"
+
+local trainTech = util.table.deepcopy(data.raw["technology"]["railway"])
```

Comment:
Good that you copy the railway technology over to make the train unlock. I would change the prerequisites, so the trainTech requires the railway and not the other prerequisintes of the railway?